dvc/stage/utils.py

```diff
@@ -57,6 +57,27 @@ def fill_stage_outputs(stage, **kwargs):
     ]
 
     stage.outs = []
+
+    dvclive_l = kwargs.get("dvclive", [])
+    if dvclive_l:
+        if len(dvclive_l) != 1:
+            from dvc.exceptions import DvcException
+
+            raise DvcException("Only one dvclive output allowed!")
+
+        from dvc.output import BaseOutput
```

Comment:
Please don't import mid-method unless you have a very good reason to do that.

Reply:
Might want to consider moving this dvclive section into a separate helper.

Reply:
created `_load_live_outputs`